IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -3,42 +3,65 @@
 import java.util.Scanner;
 
 /**
- * TODO: Description of your class.
  *
- * @author TODO: Your name here.
+ * @author Sameen Qaiser
  */
 public class IntroducingMethods {
 
+    /**
+     * @param args the command line arguments
+     */
     public static void main(String[] args) {
-
-        // TODO: Fill in the body with your code
+// TODO: Fill in the body with your code
+        Scanner keyboard = new Scanner(System.in);
+        System.out.print("Enter the string: ");
+        getInputString(keyboard);
     }
 
-    /**
-     * Given a Scanner, prompts the user for a String.  If the user enters an empty
-     * String, reports an error message and asks for a non-empty String.  Returns the
-     * String to the calling program.
-     * @param inScanner The Scanner to use to prompt the user.
-     * @return The resulting String from the user.
-     */
     private static String getInputString(Scanner inScanner) {
         // TODO: Fill in the body
+        String s = inScanner.nextLine();
+        if (s.length() > 0) {
+            getWordCount(s);
+        } else {
+            System.out.println("ERROR - string can not be empty.");
+            System.out.print("Enter the string: ");
+            s = inScanner.nextLine();
+            getWordCount(s);
+
+        }
 
-        // NOTE: Do not declare a Scanner in the body of this method.
+        return s;
```

Comment:
:-1: 
I have found a failed test case here. If I enter `"\n\nthe quick brown fox jumped\n"`, I am expecting `""the quick brown fox jumped"`. However, I receive `""`, which is the empty `String`.